demo/sample_azure_iot_embedded_sdk_pkcs11/common/azure_iot_config.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZURE_IOT_CONFIG_H
+#define AZURE_IOT_CONFIG_H
+
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Include logging header files and define logging macros in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
+ * the logging configuration for AzureIoT middleware.
+ * 3. Include the header file "logging_stack.h", if logging is enabled for AzureIoT middleware.
+ */
+
+#include "logging_levels.h"
```

Comment:
should this be changed to the new paradigm or is it good the way it is?

Reply:
Idea is in sample, we want to show how to configure our log using FreeRTOS log.